src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -115,7 +115,7 @@ then
 		echo Zipping existing 'node_modules' folder ...
 		# Make the contents of the node_modules folder appear in the zip file, not the folder itself
 		cd node_modules
-		tar -zcf ../$zippedModulesFileName .
+		{{ CompressNodeModulesCommand}} ../$zippedModulesFileName .
```

Comment:
`{{ CompressNodeModulesCommand}}` => `{{ CompressNodeModulesCommand }}` (is the blank space required? i am not sure, so asking)

Reply:
adding the space for consistency